### PR TITLE
3 small commits for 3 different tasks

### PR DESCRIPTION
**Pull request by bcalvignac**

### CODE REVIEW

File: Gcpe.Hub.API/Helpers/MappingProfile.cs [hunk 1]
```diff
@@ -11,9 +11,7 @@ public MappingProfile()
         {
             CreateMap<Activity, Models.Activity>()
                 .ForMember(dest => dest.MinistriesSharedWith, opt => opt.MapFrom(src => src.ActivitySharedWith.Select(sw => sw.Ministry.Key)))
-                .ForMember(dest => dest.Categories, opt => opt.MapFrom(src => src.ActivityCategories.Select(ac => ac.Category.Name)))
-                .ForMember(dest => dest.StartDateTime, opt => opt.MapFrom(src => DateTime.SpecifyKind((DateTime)src.StartDateTime, DateTimeKind.Local)))
```
Comment (brianorwhatever) [inline Gcpe.Hub.API/Helpers/MappingProfile.cs]: why aren't these datetime mappings needed anymore? I was looking at these when I was trying to debug the tests so am curious

File: Gcpe.Hub.API/Startup.cs [hunk 2]
```diff
@@ -40,7 +40,10 @@ public void ConfigureServices(IServiceCollection services)
 
 
             services.AddMvc()
-                .AddJsonOptions(opt => opt.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore)
+                .AddJsonOptions(opt => {
+                    opt.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
+                    opt.SerializerSettings.DateTimeZoneHandling = DateTimeZoneHandling.Local;
```
Comment (brianorwhatever) [inline Gcpe.Hub.API/Startup.cs]: ohhh this might be the answer to my question above?

File: Gcpe.Hub.API/Helpers/MappingProfile.cs (hunk 1 above)
Comment (brianorwhatever) [reply Gcpe.Hub.API/Helpers/MappingProfile.cs]: nevermind i found the answer below

**Comment by bcalvignac** [comment]
yes, it`s in 1 of my commit messages

**Pull request merged**